lib/components/related_artists/index.js

```diff
@@ -16,24 +17,30 @@ class RelatedArtists extends React.Component {
 
   constructor(props) {
     super(props)
-    this.state = this.layoutState()
+    this.state = {
+        columns: 0,
+        imageSize: {
+          width: 1,
```

Comment:
`opaque_image_view` needs some sort of aspect ratio or dimensions to render, so I start with 1.